### PR TITLE
Updated to Microsoft.Azure.Common.Authentication 1.0.16-preview

### PR DESCRIPTION
**Pull request by amarzavery**

### CODE REVIEW

**Comment by AzureRT** [comment]
Can one of the admins verify this patch?

**Comment by amarzavery** [comment]
Fixes #244

File: src/Common/Commands.ScenarioTest/Commands.ScenarioTest.csproj [hunk 1]
```diff
@@ -174,6 +174,7 @@
     <Compile Include="Common\TestEnvironment.cs" />
     <Compile Include="Common\TestEnvironmentFactory.cs" />
     <Compile Include="Common\TokenCloudCredentialsHelper.cs" />
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/Common/Commands.ScenarioTest/Commands.ScenarioTest.csproj]: delete this

File: src/ResourceManager/Tags/Commands.Tags/Commands.Tags.csproj [hunk 2]
```diff
@@ -136,6 +136,7 @@
       <SubType>Designer</SubType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ResourceManager/Tags/Commands.Tags/Commands.Tags.csproj]: delete this

File: src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj [hunk 3]
```diff
@@ -115,6 +115,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj]: delete this

File: src/ServiceManagement/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj [hunk 4]
```diff
@@ -164,6 +164,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj]: delete this

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Extensions.Test/Commands.ServiceManagement.Extensions.Test.csproj [hunk 5]
```diff
@@ -141,6 +141,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Extensions.Test/Commands.ServiceManagement.Extensions.Test.csproj]: delete this

File: src/ServiceManagement/Compute/Commands.ServiceManagement.PlatformImageRepository/Commands.ServiceManagement.PlatformImageRepository.csproj [hunk 6]
```diff
@@ -183,6 +183,7 @@
     <Content Include="PIR.psd1">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.PlatformImageRepository/Commands.ServiceManagement.PlatformImageRepository.csproj]: delete this

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Preview/Commands.ServiceManagement.Preview.csproj [hunk 7]
```diff
@@ -162,6 +162,7 @@
     <Compile Include="ServiceManagementPreviewProfile.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Preview/Commands.ServiceManagement.Preview.csproj]: delete this

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Commands.ServiceManagement.Test.csproj [hunk 8]
```diff
@@ -418,6 +418,7 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="FunctionalTests\ExtensionTests\MicrosoftAntimalware\AzureVMMicrosoftAntimalwareExtensionFunctionalTests.ps1" />
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Commands.ServiceManagement.Test.csproj]: delete this

File: src/ServiceManagement/Compute/Commands.ServiceManagement/Commands.ServiceManagement.csproj [hunk 9]
```diff
@@ -490,6 +490,7 @@
     <Compile Include="StorageServices\StorageCredentialsFactory.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement/Commands.ServiceManagement.csproj]: and this

File: src/ServiceManagement/HDInsight/Commands.HDInsight.Test/Commands.HDInsight.Test.csproj [hunk 10]
```diff
@@ -232,6 +232,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/HDInsight/Commands.HDInsight.Test/Commands.HDInsight.Test.csproj]: and this

File: src/ServiceManagement/ManagedCache/Commands.ManagedCache.Test/Commands.ManagedCache.Test.csproj [hunk 11]
```diff
@@ -146,6 +146,7 @@
     <Compile Include="ScenarioTests\ManagedCacheTestsBase.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/ManagedCache/Commands.ManagedCache.Test/Commands.ManagedCache.Test.csproj]: and this

File: src/ServiceManagement/ManagedCache/Commands.ManagedCache/Commands.ManagedCache.csproj [hunk 12]
```diff
@@ -145,6 +145,7 @@
     <Compile Include="Service\RemoveAzureManagedCacheNamedCache.cs" />
     <Compile Include="Service\SetAzureManagedCache.cs" />
     <Compile Include="Service\SetAzureManagedCacheNamedCache.cs" />
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/ManagedCache/Commands.ManagedCache/Commands.ManagedCache.csproj]: and this

File: src/ServiceManagement/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 13]
```diff
@@ -146,6 +146,7 @@
     <Compile Include="ScenarioTests\NetworkTestsBase.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (yugangw-msft) [inline src/ServiceManagement/Network/Commands.Network.Test/Commands.Network.Test.csproj]: and this

**Comment by yugangw-msft** [comment]
i think i know the reason, that we should not remove the "app.config" for the "Commands.StorSimple.Test.csproj" which does define legit app configuration values

**Pull request merged**